README.md

```diff
@@ -42,5 +50,19 @@ class MyComponent extends React.Component {
 }
 ```
 
+#### Limiting Highchart Rerenders
+Rerendering a highcharts graph is expensive. You can pass in a `isPureConfig` option to the `Highcharts` component, which will keep the highcharts graph from being updated so long as the provided `config` is [referentially equal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators) to its previous value.
+
 ## Using highcharts-more
 Just require `require('react-highcharts/more');` instead of `react-highcharts`
+
+## Update Highcharts Version
+
+ 1. Install new highcharts version.
+    ```
+    npm unistall highcharts-release --save-dev
+    npm install highcharts-release --save-dev
+    ```
+    
+ 2. Increment the version such that a patch, minor release, or major release of 
```

Comment:
Yep, its just a friendly reminder to contributors. It's easy to think of pulling a new highcharts version as a minor change when in fact it can be a major update.